README.md

```diff
@@ -1,18 +1,40 @@
-## FOSSUnited Platform
+<div align="center">
+    <img alt="fossunited logo" src="https://raw.githubusercontent.com/fossunited/Branding/main/asset/general-logos/logo-white.png" width="150px" height="120px">
+</div>
 
-Built on Frappe
+<div align="center">
+<img alt="Static Badge" src="https://img.shields.io/badge/Builton-Frappe-blue?style=for-the-badge&labelColor=%231a1a1a&color=blue&link=https%3A%2F%2Ffrappe.io"/>
 
-## Contributing
+<img alt="static badge" src="https://img.shields.io/badge/License-AGPL-%2308b47f?style=for-the-badge&labelColor=%231a1a1a&color=%2308b47f&link=https%3A%2F%2Fwww.gnu.org%2Flicenses%2Fagpl-3.0.en.html"/>
+</div>
 
-### Pre Commit Hooks
+## The FOSS United Platform 
+Repo for the website and open-source platform of FOSS United. The whole platform is being built on [Frappe](https://frappe.io). 
 
+## Installation 
+- This project works the best on the latest Frappe Version v15. And is suggested to be installed on the same. 
+- Checkout Frappe Framework [Installation Docs](https://frappeframework.com/docs/) for setting up frappe on your [bench](https://frappeframework.com/docs/user/en/tutorial/install-and-setup-bench). 
+
+- Create a new bench with 
+```bench fossu-bench``` 
+- Clone the FOSS United Platform App. 
+```bench get-app https://github.com/fossunited/fossunited```
+- Create a [new site](https://frappeframework.com/docs/user/en/tutorial/create-a-site) 
+```bench new-site test.localhost```
+- Install the App on the created site 
+```bench --site test.localhost install-app fossunited```
+- Finally, 
+```bench start```
```

Comment:
- Rather than telling them each and every step to set up a bench, let's just link the Frappe Docs for bench installation and setup. And then start from bench get-app.
- Also, you have to get app and install on a new site with bench still running.

Reply:
1. I don't see any bench installation steps there. It starts from creating a bench directory. And yeah, I've already linked Frappe Docs for installing Frappe. 
2. I remember, when I used to do this. My site used to get stuck on that "update" message page and I had to kill all the services running the bench.